lib/browser/methods/devices.rb

```diff
@@ -37,5 +37,9 @@ def playbook?
     def windows_touchscreen_desktop?
       windows? && !!(ua =~ /Touch/)
     end
+    
+    def desktop?
+      !mobile? && !tablet?
```

Comment:
What if user agent is a `console?`. https://github.com/craigsheen/browser/blob/add_desktop_method/lib/browser/methods/consoles.rb#L29